stub/include/neurala/utils/string.h

```diff
@@ -338,6 +338,22 @@ toUpperCase(const std::string& str)
 	return result;
 }
 
+NEURALA_PUBLIC inline bool
```

Comment:
do you need `NEURALA_PUBLIC` for inline functions?

Reply:
Not really, I think it's actually a bad pattern since it creates a standalone addressable copy in case another module needs a pointer to it, even if it's unused.
But it still allows another modules to inline it too

Reply:
I merged. Open a new PR to remove if needed